src/liberty/libertyProject.ts

```diff
@@ -132,38 +133,61 @@ export class ProjectProvider implements vscode.TreeDataProvider<LibertyProject>
 				}
 			}).catch((err: any) => console.error("Unable to parse build.gradle: " + gradlePath + "; " + err));
 		}
-
 		return validGradleBuildFiles;
 	}
 
+	private projectRootPathExists(path: string, keys: Iterable<string>): Boolean {
+		for (const existingPath of keys) {
+			if ( vscodePath.dirname(existingPath) === path) {
+				return true;
+			}
+		  }
+		return false;
+	}
+
+	/**
+	 * Checks if given build file exists in existing project map (<code>this.projects</code>).
+	 * If it exists, then the corresponding liberty project will be added to the given 
+	 * <code>projectsMap</code>.
+	 * 
+	 * @param buildFilePath The build file to check (full path to pom.xml or build.gradle)
+	 * @param projectType Project type
+	 * @param projectsMap Existing projects.
+	 * @returns 
+	 */
+	private addExistingProjectToNewProjectsMap (buildFilePath: string, projectType: string, projectsMap: Map<string, LibertyProject> ) : Boolean {
+		let added = false;
+		if (this.projects.has(buildFilePath)) {
+			// check version of liberty-maven-plugin to see if it is valid for contatiners
```

Comment:
```suggestion
```

Reply:
This comment is misleading as the check for this is here: https://github.com/OpenLiberty/liberty-tools-vscode/pull/127/files#diff-5f236a0e97f15adc9719084cb9e2e4fe818e88043da51ff05c25d585e27eec26R179-R180